packages/manifest-schemas/formats/web3api/0.0.1-prealpha.6.json

```diff
@@ -9,34 +9,42 @@
   ],
   "properties": {
     "format": {
+      "description": "Polywrap YAML format version.",
       "type": "string",
       "const": "0.0.1-prealpha.6"
     },
     "build": {
+      "description" : "Path to customized build YAML file.",
```

Comment:
```suggest
      "description": "Path to the custom build manifest file.,
```

This can be either JSON/YAML file.
https://github.com/polywrap/monorepo/blob/19eb5db42e66848f3236d694983293feb4a52676/packages/js/core/src/manifest/validators.ts#L90-L92

Reply:
Updated the description.

Reply:
It should be `manifest` instead of `Yaml` since we allow both Yaml and JSON format here.

Reply:
Replaced `YAML` with  `manifest`